app/views/ctc/questions/file_full_return/edit.html.erb

```diff
@@ -6,14 +6,12 @@
 
   <p><%= t("views.ctc.questions.file_full_return.help_text", current_tax_year: current_tax_year)%></p>
 
-  <div class="reveal-shrink-wrapper">
```

Comment:
This was just a random thing I came across where there was a full width button on the page and then this reveal that was awkwardly shorter than the button. Removing this class makes it the same width as the button.